.github/workflows/action.yml

```diff
@@ -6,7 +6,28 @@ on:
       - "www/**"
 
 jobs:
+  format:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Get changed files
+        id: changed-files
+        uses: jitterbit/get-changed-files@v1
+        continue-on-error: true
+
+      - name: Prettier files
+        uses: actions/checkout@v2.3.5
+        with:
+          ref: ${{ github.head_ref }}
+      - run: npm i prettier -g
+      - run: prettier --write ${{ steps.changed-files.outputs.all }}
+
+      - name: Commit changes
+        uses: stefanzweifel/git-auto-commit-action@v4
+        with:
+          commit_message: "fix(code): prettier"
+          branch: ${{ github.head_ref }}
```

Comment:
thought: Instead of committing directly to the branch, should we fail and block merging to avoid any nasty surprises?

Reply:
I've seen pipelines generate linting reports. Maybe overkill for now, but would be a good dev. experience